README.md

```diff
@@ -84,3 +84,51 @@ For example, if we wanted to exclude the copyright/license check for a directory
   }
 }
 ```
+
+# Creating a Whitelist for Copyright/License Check in Repolinter
```

Comment:
The single `#` heading should only be used once in a md file (title), can we have this match the previous section, i.e. `###`?